_specs/ecip-1000.md

```diff
@@ -293,64 +293,137 @@ To avoid doubt: comments and status are unrelated metrics to judge an ECIP, and
 
 ### Specification
 
-New ECIPs may be accepted with the following licenses. Each new ECIP must identify at least one acceptable license in its preamble. The License header in the preamble must be placed after the Created header. Each license must be referenced by their respective abbreviation given below.
+All new ECIPs should use Apache 2.0 or newer licensing for the specification and
```

Comment:
All code contributed to a repository assumes the repositories license. Best to let legal precedent do the talking, and not even mention license at all.